### ISSUE TITLE
Rootkit

### ISSUE THREAD

**Issue by Samwadihol** [UnamSanctam/SilentCryptoMiner#925]
hey hi, thanks for the update, i wanted to try the rootkit again after a long time, but i guess only etc works, both of them seem to work in checker, but only etc comes to my panel

**Comment by Samwadihol** [comment]
no it comes to my panel when i disable it

**Comment by Samwadihol** [comment]
this problem was only happening with rtm i guess i'm not sure about it i will replace it with monero instead of rtm and try again

**Comment by Samwadihol** [comment]
but when I turn off the rootkit, it appears on the panel

**Comment by UnamSanctam** [comment]
Yes but it could still be due to the pool, when I try mining to flockpool with most miners (when testing other miners) I have to restart them multiple times to get flockpool to connect. During my latest attempt after restarting it 10 times it worked 2 out of those restarts (20% of the time).

**Comment by Samwadihol** [comment]
https://user-images.githubusercontent.com/108399515/177748928-bc81aee2-8039-4440-90b1-b7d41ff2b0f7.mp4

**Comment by Samwadihol** [comment]
ah also there seems to be a little trouble here haha

**Comment by Samwadihol** [comment]
I made monero instead of rtm, I'll try again

**Comment by Samwadihol** [comment]
I changed it to monero now and I'm waiting for it to come to my panel

**Comment by Samwadihol** [comment]
![image](https://user-images.githubusercontent.com/108399515/177749666-1617248b-061b-4dce-950b-ed78f105471d.png)

**Comment by Samwadihol** [comment]
when i made the pool moneroocean it looked like this on the panel

**Comment by UnamSanctam** [comment]
It means that the pool connection settings are wrong and thus it can't connect.

**Comment by Samwadihol** [comment]
![image](https://user-images.githubusercontent.com/108399515/177749928-9a9cab75-451b-4879-b448-cf206d655f06.png)

**Comment by Samwadihol** [comment]
It looks like this after waiting for a while

**Comment by Samwadihol** [comment]
> It means that the pool connection settings are wrong and thus it can't connect.

no its not

**Comment by UnamSanctam** [comment]
> no its not

It's says "Error" if there's no active pool currently, which means that it couldn't connect (either settings are wrong or the pool refused your connection).

> It looks like this after waiting for a while

Then that is working.

**Comment by Samwadihol** [comment]
so if it shows up in checker even if it doesn't show in my panel, does it mean that the miner is working?

**Comment by UnamSanctam** [comment]
Most likely, or the pool connection gives an error (usually happens with flockpool) which means the miner itself never really starts (which is why it wouldn't appear in the panel).

**Comment by Samwadihol** [comment]
thank you for your help i will send you 10$ etc as a small amount but
get it haha

**Comment by UnamSanctam** [comment]
Haha thank you, good luck with your mining.
